pages/api/retrieval.ts

```diff
@@ -0,0 +1,170 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import isRequestAllowed from "@lib/middleware/httpRequestAllowed";
+import { logMessage } from "@lib/logger";
+import { DynamoDBClient, QueryCommand, UpdateItemCommand } from "@aws-sdk/client-dynamodb";
+import { FormUserDBConfigProperties, BearerTokenPayload } from "@lib/types";
+import { getBearerToken } from "@lib/middleware/bearerToken";
+import executeQuery from "@lib/integration/queryManager";
+import dbConnector from "@lib/integration/dbConnector";
+import { QueryResult } from "pg";
+import jwt from "jsonwebtoken";
+
+/**
+ * Request type: GET
+ * USAGE: 
+ * curl http://localhost:3000/api/retrieval?maxRecords=10&formID=1
+   -H "Accept: application/json"
+   -H "Authorization: Bearer {token}"
+ * 
+ * This method fetchs form responses up to the maximum number (maxRecords) that was specified.
+ * It required to have a valid bearer token which must carry a valid formID. When all these conditons are met,
+ * Not only it should return form reponses but also mark those responses as retrieved by updating the attribute Retrieved
+ * from 0 to 1.
+ * @param res 
+ * @param formID 
+ * @param maxRecords 
+ * @returns 
+ */
+export async function getFormResponses(
```

Comment:
Since this is exported as `default` on Line 170, it doesn't need to be exported here.